<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VerifyForm</class>
 <widget class="QWidget" name="VerifyForm">
  <layout class="QVBoxLayout">

   <item><widget class="QLabel" name="smsLabel"/></item>

   <item><widget class="QLineEdit" name="smsEdit">
     <property name="placeholderText"><string>Введите SMS код</string></property>
   </widget></item>

   <item><widget class="QPushButton" name="smsBtn">
     <property name="text"><string>Проверить SMS</string></property>
   </widget></item>

   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Пациент 65 лет, АД 170/100, глюкоза 8.5 — диагноз?</string>
     </property>
    </widget>
   </item>

   <item><widget class="QLineEdit" name="capEdit">
     <property name="placeholderText"><string>Ответ</string></property>
   </widget></item>

   <item><widget class="QPushButton" name="capBtn">
     <property name="text"><string>Проверить</string></property>
   </widget></item>

  </layout>
 </widget>
</ui>
